RHDEVDOCS-6198: Content creation for PSA labels note (#85061)
openshift/openshift-docs: argocd_instance/setting-up-argocd-instance.adoc

[NOTE]
====
For {gitops-shortname} version 1.13 and later:

* Route TLS termination is set as default to the `reencrypt` mode for both the default and user-defined Argo CD instances. TLS connections to the Argo CD instances now receive the default ingress certificate that is set in {OCP}, instead of the self-signed Argo CD certificate. You can modify the route TLS termination policy by configuring the `.spec.server.route.tls` field of the Argo CD CR.

> * Restrictive Pod Security Admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads, for example - monitoring or logging, in this namespace, ensure they comply with the restrictive PSA requirements. If compliance is not feasible, consider using a user-defined, cluster-scoped ArgoCD instance, where PSA labels are not applied or are controlled by the operator.

Reviewer: - The profile is called "restricted", not "restrictive", so I assume you'd want to use that term? https://docs.openshift.com/container-platform/4.17/authentication/understanding-and-managing-pod-security-admission.html#psa-profiles_understanding-and-managing-pod-security-admission
- Pod security admission isn't capitalized
- When you say "the Operator", can you clarify which Operator you mean? 
- A few other suggestions

```suggestion
* Restricted pod security admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads in this namespace, such as monitoring or logging, ensure that they comply with the restricted PSA requirements. If compliance is not feasible, consider using a user-defined, cluster-scoped ArgoCD instance, where PSA labels are not applied or controlled by the Operator.
```
